test/playlist-loader.test.js

```diff
@@ -1478,6 +1478,78 @@ QUnit.test('clears the update timeout when switching quality', function(assert)
   assert.equal(1, refreshes, 'only one refresh was triggered');
 });
 
+QUnit.test('Increments retries on playlist error', function(assert) {
+  const loader = new PlaylistLoader('manifest/master.m3u8', this.fakeVhs);
+
+  loader.load();
+
+  // master
+  this.requests.shift().respond(
+    200, null,
+    '#EXTM3U\n' +
+                                '#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=17\n' +
+                                'playlist/playlist.m3u8\n' +
+                                '#EXT-X-STREAM-INF:PROGRAM-ID=2,BANDWIDTH=170\n' +
+                                'playlist/playlist2.m3u8\n' +
+                                '#EXT-X-ENDLIST\n'
```

Comment:
Some weird whitespace here
```suggestion
    '#EXTM3U\n' +
    '#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=17\n' +
    'playlist/playlist.m3u8\n' +
    '#EXT-X-STREAM-INF:PROGRAM-ID=2,BANDWIDTH=170\n' +
    'playlist/playlist2.m3u8\n' +
    '#EXT-X-ENDLIST\n'
```